nexus/tests/integration_tests/instances.rs

```diff
@@ -4280,6 +4289,102 @@ async fn test_updating_missing_instance_is_not_found(
     );
 }
 
+async fn expect_instance_reconfigure_ok(
+    external_client: &ClientTestContext,
+    instance_id: &Uuid,
+    update: params::InstanceUpdate,
+) -> Instance {
+    let url_instance_update = format!("/v1/instances/{instance_id}");
+
+    let builder = RequestBuilder::new(
+        external_client,
+        http::Method::PUT,
+        &url_instance_update,
+    )
+    .body(Some(&update))
+    .expect_status(Some(http::StatusCode::OK));
+
+    let response = NexusRequest::new(builder)
+        .authn_as(AuthnMode::PrivilegedUser)
+        .execute()
+        .await
+        .expect("can attempt to reconfigure the instance");
+
+    response
+        .parsed_body::<Instance>()
+        .expect("response should be parsed as an instance")
+}
+
+// Test reconfiguring an instance's auto-restart policy.
+#[nexus_test]
+async fn test_auto_restart_policy_can_be_changed(
+    cptestctx: &ControlPlaneTestContext,
+) {
+    let client = &cptestctx.external_client;
+    let instance_name = "reincarnation-station";
+
+    create_project_and_pool(&client).await;
+
+    let instance_params = params::InstanceCreate {
+        identity: IdentityMetadataCreateParams {
+            name: instance_name.parse().unwrap(),
+            description: String::from("stuff"),
+        },
+        ncpus: InstanceCpuCount::try_from(2).unwrap(),
+        memory: ByteCount::from_gibibytes_u32(4),
+        hostname: instance_name.parse().unwrap(),
+        user_data: vec![],
+        ssh_public_keys: None,
+        network_interfaces: params::InstanceNetworkInterfaceAttachment::Default,
+        external_ips: vec![],
+        boot_disk: None,
+        disks: Vec::new(),
+        start: false,
```

Comment:
```suggestion
        start: true,
```
should work, and also should keep a well-intentioned dork (me) from accidentally disallowing this with a future refactor

Reply:
yeah, good call.